phone_actions.rb

```diff
@@ -0,0 +1,15 @@
+class PhoneActions
+
+  def initialize(username, password)
+    @actions_url = "https://#{username}:#{password}@api.<made_up>.com/actions/"
+  end
+
+  def call!(number, message)
+    JSON.parse(RestClient.post("@actions_url/#{text}", {message: message, number: number}))
```

Comment:
```suggestion
    JSON.parse(RestClient.post("#{@actions_url}/#{text}", {message: message, number: number}))
```

On both of these calls you forgot to wrap `@actions_url` with the string parameter syntax.